lib/domain/usecase/room/upload_avatar_new_group_chat_interactor.dart

```diff
@@ -0,0 +1,34 @@
+
+import 'package:dartz/dartz.dart';
+import 'package:fluffychat/app_state/failure.dart';
+import 'package:fluffychat/app_state/success.dart';
+import 'package:fluffychat/domain/app_state/room/upload_avatar_new_group_chat_state.dart';
+import 'package:flutter/foundation.dart';
+import 'package:matrix/matrix.dart';
+
+class UploadAvatarNewGroupChatInteractor {
```

Comment:
should be a common for uploading media.
```suggestion
class UploadContentInteractor {
```